src/modules/components/member/memberDataListItem/memberDataListItemStructure/memberDataListItemStructure.tsx

```diff
@@ -15,7 +16,7 @@ export interface IMemberDataListItemProps extends IDataListItemProps {
     /**
      * The total voting power of the member.
      */
-    votingPower?: number;
+    votingPower?: number | string;
```

Comment:
I expected string vp to be pre-formatted coming in, but then we are formatting it again..Would it make sense if it's a string to set the original string as the fallback value when formatting, since we've now no guarantee that the string can be parsed into a number? This might just be overcomplicating things though...

Reply:
Mm.. well we'll always get such parameters as strings from the backend (votingPower, balance, etc..), I believe we should keep handling the string types to avoid converting such strings to numbers on the apps

Reply:
No getting it back as a string is fine. I'm thinking about the extensive external users assuming that this can be any string and not just a string that can be parsed into a number.

Reply:
Oh got it, yes it might be confusing for other devs. Mm.. maybe we can handle both formatted & unformatted strings in the future or just write a doc about such params